kobo/apps/subsequences/integrations/google/base.py

```diff
@@ -0,0 +1,128 @@
+from abc import ABC, abstractmethod
+from concurrent.futures import TimeoutError
+from typing import Any, Tuple
```

Comment:
`Tuple` is deprecated. Use `from __future__ import annotations` and use lowercase type everywhere instead.
See https://docs.python.org/3/library/typing.html#deprecated-aliases